pages/linux/smem.md

```diff
@@ -0,0 +1,20 @@
+# smem
+
+> Show memory usage for programs.
+> More information: <https://manned.org/smem>.
+
+- Print memory usage for current processes:
+
+`smem`
+
+- Print memory usage for current processes for a every user on a system:
+
+`smem --users`
+
+- Print memory usage for current processes for a specified user:
```

Comment:
```suggestion
- Print memory usage for all current processes for a specific user:
```

Reply:
From what I can tell, it is not all processes